### ISSUE TITLE
Feature request: Enable Login into a Web Application

### ISSUE THREAD

**Issue by felixAnhalt** [EU-EDPS/website-evidence-collector#107]
**Description:**

I would like to propose the addition of a new feature to the Website Evidence Collector tool that allows users to automate the login process into a web application. This enhancement would be beneficial for scenarios where evidence collection involves authenticated sessions.

**Feature Details:**

Description: Enable the tool to perform automated login actions on a specified web application.
Use Case: Users sometimes need to collect evidence from authenticated areas of websites, and automating the login process would streamline this workflow.
Implementation Approach: Consider using the Puppeteer library to interact with login forms and credentials via css selectors.

**Additional Context:**

We already built this for our own use cases with an implementation based on environment variables.
I'm not sure wether or not this approach is the best in the bigger context of the application, but it feels more natural to add the necessary variables like login selector and username via env instead of having four more configuration options for the cli.
I'll provide a pull request referencing this issue.

**Comment by rriemann** [comment]
I agree that having a solution here would be useful. Now, all login forms are a bit different. So Let's assume you need to study first the login mechanism and then customize the setting of WEC accordingly.

Would this be really easier than creating manually a Chrome profile folder and use Chrome to login already, before loading this profile folder for the evidence collection?

**Comment by felixAnhalt** [comment]
As a full-stack developer integrating the tool within a Docker container for seamless invocation within a larger web application, it seems intuitive to introduce four environment variables. I have documented the constraints imposed by captchas and other login procedures, which operate independently of the conventional username/password mechanism, in the README.

I think we should also be considering the possibility of implementing SSO, but I'd argue the traditional username/password flow is used enough to justify the existence of this feature.
